src/utilities/CMakeLists.txt

```diff
@@ -44,13 +57,15 @@ if(MANIFOLD_PAR STREQUAL "TBB")
         include(FetchContent)
         set(TBB_TEST OFF CACHE INTERNAL "" FORCE)
         set(TBB_STRICT OFF CACHE INTERNAL "" FORCE)
+        set(BUILD_SHARED_LIBS ON)
         FetchContent_Declare(TBB
-          GIT_REPOSITORY https://github.com/oneapi-src/oneTBB.git
-          GIT_TAG        v2021.10.0
-          GIT_SHALLOW    TRUE
-          GIT_PROGRESS   TRUE
+            GIT_REPOSITORY https://github.com/oneapi-src/oneTBB.git
+            GIT_TAG        v2021.10.0
+            GIT_SHALLOW    TRUE
+            GIT_PROGRESS   TRUE
```

Comment:
originally I wanted to use `find_package` for TBB as well (in addition to pkg-config), but it seems that the `FindTBB.cmake` provided in some versions are broken, and calling `find_package(tbb)` will cause cmake error, so I just keep the old behavior.